frontend/src/pages/modelServing/screens/projects/ServingRuntimeModal/ServingRuntimeReplicaSection.tsx

```diff
@@ -16,13 +16,16 @@ const ServingRuntimeReplicaSection: React.FC<ServingRuntimeReplicaSectionProps>
     const target = event.target as HTMLInputElement;
     setData('numReplicas', parseInt(target.value));
   };
+  const validate = data.numReplicas >= 0 ? ValidatedOptions.default : ValidatedOptions.error;
```

Comment:
This will never be an error? Your field doesn't allow for negative numbers. In fact, it's min is 1 -- not even 0. I'm confused about this logic.

Reply:
Best guess is improper data originally?

Reply:
No, this is targeted for when the user input the fields manually, not using the counter buttons. This  was a feedback from QE, you can trigger an empty value/0 as an input field.

Reply:
Sounds like you need smarter onChange logic... I thought the number input just brought it back to the min value -- if you can actually input anything, it's not a valid value and it should snap back to `min`.

Showing something wrong serves nobody -- unless you can think of a flow a user can do that starts invalid and goes to valid... something like 1.2 "1" valid, "1." invalid, "1.2" valid. But I'm not sure that is the case here. Letting them type "abc" in a field intended for a number is weird.

Reply:
Mmmm yeah, I am not happy with the result either, I tried first adding more logic for the onChange handler but I bumped into some issues too. I'm gonna create an issue to cover this specific logic. Thanks again.